### PR TITLE
feat(*): added register button in login page

### PR DESCRIPTION
**Pull request by amin-newcastle**

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -6,7 +6,8 @@
          <%= text_field_tag :email %><br><br>
          <%= label_tag :password%>
          <%= password_field_tag :password%><br><br>
-         <%= submit_tag "Log In"%>
+         <%= submit_tag "Log In", class: 'welcome-button'%>
```
Comment (lewiscj97) [inline app/views/sessions/new.html.erb:9]: Can these buttons be the same as the ones on the timesheet page please (the smaller grey ones)

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
       <%end%>
+      <%= button_to "Register", '/users/new', method: :get, class: 'welcome-button' %><p>if you don't have an account</p>
```
Comment (lewiscj97) [inline app/views/sessions/new.html.erb:11]: Can you capitalise the `if you don't...` please, and also put the p tag on it's own line above the button?

**Review by lewiscj97 — CHANGES_REQUESTED** [review]
Looks good - just a few stylistic changes. Could you also write a quick feature test for clicking the button and asserting that it takes you to the `Log In` path?

File: app/views/sessions/new.html.erb (hunk 2 above)
Comment (amin-newcastle) [reply app/views/sessions/new.html.erb:11]: ALL uppercase? IF YOU DON'T...

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (amin-newcastle) [reply app/views/sessions/new.html.erb:9]: Done

**Comment by amin-newcastle** [comment]
Also wrote feature test

**Review by lewiscj97 — APPROVED** [review]

**Pull request merged**